### PR TITLE
chore: Update tracing & metrics

### PR DESCRIPTION
**Pull request by viccuad**
## Description

This updates the tracing and metrics how-tos so they are easier to apply.

## Test

Tested by following the docs. Tracing and Prometheus seem to work Grafana doesn't. But maybe I misconfigured the `kube-prometheus-stack-values.yaml` and that's why I'm not receiveing enough metrics for Grafana.

### Potential improvement

Feel free to edit the commits as needed.

### CODE REVIEW

File: docs/howtos/telemetry/20-tracing-qs.md [hunk 1]
```diff
@@ -328,3 +328,7 @@ as there is a new service `kubewarden-policy-server` listed in the UI:
 ![Jaeger dashboard](/img/jaeger-ui-home.png "The dashboard of Jaeger")
 
 The Jaeger collector is properly receiving the traces generated by our PolicyServer.
+
+For accessing the Jaeger UI, you can use create an ingress or use
+`kubectl -n jaeger port-forward service/my-open-telemetry-query 16686`
+And go to [localhost:16686](localhost:16686).
```
Comment (jhkrug) [inline docs/howtos/telemetry/20-tracing-qs.md:334]:
```suggestion
To access the Jaeger UI, you can create an ingress or use
`kubectl -n jaeger port-forward service/my-open-telemetry-query 16686`
then go to [localhost:16686](localhost:16686).
```

File: docs/howtos/telemetry/30-metrics-qs.md [hunk 2]
```diff
@@ -31,7 +31,36 @@ that allows us to define Prometheus' Targets intuitively.
 There are many ways to install and set up Prometheus. For ease of deployment, we will use the
 Prometheus community Helm chart.
 
-Let's install the Prometheus stack Helm Chart:
+Let's create a `kube-prometheus-stack-values.yaml` file with the
+following contents:
+
+````yaml
```
Comment (jhkrug) [inline docs/howtos/telemetry/30-metrics-qs.md:37]:
```suggestion
```yaml
```

File: docs/howtos/telemetry/30-metrics-qs.md [hunk 3]
```diff
@@ -221,3 +252,5 @@ variable to match the name of the desired policy.
 You should be able to see the dashboard similar to this:
 
 ![Dashboard](/img/grafana_dashboard.png)
+````
```
Comment (jhkrug) [inline docs/howtos/telemetry/30-metrics-qs.md:255]:
```suggestion
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
A minor change and I think some additional backticks crept in, looking at the preview:

https://deploy-preview-431--silly-bunny-8cedd0.netlify.app/next/howtos/telemetry/metrics-qs

File: docs/howtos/telemetry/30-metrics-qs.md [hunk 4]
```diff
@@ -31,7 +31,36 @@ that allows us to define Prometheus' Targets intuitively.
 There are many ways to install and set up Prometheus. For ease of deployment, we will use the
 Prometheus community Helm chart.
 
-Let's install the Prometheus stack Helm Chart:
+Let's create a `kube-prometheus-stack-values.yaml` file with the
+following contents:
+
+````yaml
+---
+prometheus:
```
Comment (fabriziosestito) [inline docs/howtos/telemetry/30-metrics-qs.md:39]: isn't this redundant with the two `ServiceMonitor` resources created below?

File: docs/howtos/telemetry/30-metrics-qs.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (viccuad) [reply docs/howtos/telemetry/30-metrics-qs.md]: Indeed, resolved with John's changes.

**Review by jhkrug — APPROVED** [review]
lgtm

File: docs/howtos/telemetry/30-metrics-qs.md (hunk 4 above)
Comment (viccuad) [reply docs/howtos/telemetry/30-metrics-qs.md]: Indeed, dropped them in https://github.com/kubewarden/docs/pull/431/commits/95ad718e5e33778e462ca1cfd5bcdc9eae8f1e66.

**Pull request merged**